exonum/src/node/mod.rs

```diff
@@ -948,8 +945,7 @@ impl Node {
     /// Creates node for the given services and node configuration.
     pub fn new(
         database: impl Into<Arc<dyn Database>>,
-        external_runtimes: impl IntoIterator<Item = impl Into<RuntimeInstance>>,
-        services: impl IntoIterator<Item = Box<dyn ServiceFactory>>,
+        with_runtimes: impl FnOnce(mpsc::Sender<UpdateEndpoints>) -> Vec<RuntimeInstance>,
```

Comment:
Will this constructor signature be improved after `api` is moved to a separate crate? IMO, it looks quite terrible. (I don't have a better solution in mind yet, but I'll try to come up with one.) Additionally, `UpdateEndpoints` in the current form cannot work with multiple runtimes (any update will override the entire set of endpoints).

Reply:
At the moment I don't know how I can make this API better. 
I may write todo comment with explanation why this API so terrible and what we should do to fix it.